script.js

```diff
@@ -0,0 +1,136 @@
+// Start here
+
+// Step 1 - Welcome and introduction
+// Your code goes here
+alert(
+  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+);
+// Asking for name
+
+  let userName = prompt("What is your name?");
+
+// follow up greeting with name
+
+if (userName) {
+  alert(`Hello ${userName}! Welcome!`);
+} else {
+  alert(`Hello there! Welcome `);
+}
+
+// Step 2 - Food choice
+
+//asking for userChoice
+let chosenPizza;
+
+while (true) {
```

Comment:
well done on this while loop and the one in line 53! they work well and I think it was outside of the scope of our project, so you pushed yourself, good job